python/semantic_kernel/connectors/ai/open_ai/services/open_ai_chat_completion_base.py

```diff
@@ -267,32 +313,45 @@ def _chat_message_content_to_dict(self, message: "ChatMessageContent") -> dict[s
     # endregion
     # region internal handlers
 
-    async def _send_chat_request(self, settings: OpenAIChatPromptExecutionSettings) -> list["ChatMessageContent"]:
+    async def _send_chat_request(
+        self, settings: OpenAIChatPromptExecutionSettings
+    ) -> list["ChatMessageContent"]:
         """Send the chat request."""
         response = await self._send_request(request_settings=settings)
         response_metadata = self._get_metadata_from_chat_response(response)
-        return [self._create_chat_message_content(response, choice, response_metadata) for choice in response.choices]
+        return [
+            self._create_chat_message_content(response, choice, response_metadata)
+            for choice in response.choices
+        ]
 
     async def _send_chat_stream_request(
         self, settings: OpenAIChatPromptExecutionSettings
     ) -> AsyncGenerator[list["StreamingChatMessageContent | None"], None]:
         """Send the chat stream request."""
         response = await self._send_request(request_settings=settings)
         if not isinstance(response, AsyncStream):
-            raise ServiceInvalidResponseError("Expected an AsyncStream[ChatCompletionChunk] response.")
+            raise ServiceInvalidResponseError(
+                "Expected an AsyncStream[ChatCompletionChunk] response."
+            )
         async for chunk in response:
             if len(chunk.choices) == 0:
                 continue
             chunk_metadata = self._get_metadata_from_streaming_chat_response(chunk)
             yield [
-                self._create_streaming_chat_message_content(chunk, choice, chunk_metadata) for choice in chunk.choices
+                self._create_streaming_chat_message_content(
+                    chunk, choice, chunk_metadata
+                )
+                for choice in chunk.choices
             ]
 
     # endregion
     # region content creation
 
     def _create_chat_message_content(
-        self, response: ChatCompletion, choice: Choice, response_metadata: dict[str, Any]
```

Comment:
There are a lot of formatting changes which don't look pertinent to the goal of the PR. Can we please revert them?

Reply:
Of course!

I think we also need to provide instructions on how to setup auto formatting.

Reply:
I see there is another .vscode in the Python subfolder. I will recommend developers to develop in the Python subfolder instead of the root.

Reply:
I have removed all the undesired formatting from my IDE.

Reply:
https://github.com/microsoft/semantic-kernel/pull/6797